README.md

```diff
@@ -1,180 +1,37 @@
-* Do not include in the description the development status of the extension (e.g. _draft_). If you need to add current status, do so in a _README_ file, it will be much more visible and therefore less prone to be forgotten and not updated.
-* Avoid descriptions that simply duplicate or paraphrase the extension name.
-
-For example, for [ocds_performance_failures_extension](https://github.com/open-contracting-extensions/ocds_performance_failures_extension), this wouldn't be a good description:
-
-> An extension covering performance failures in OCDS.
-
-The actual description in the extension is much better:
-
-> Adds fields to the implementation section to allow disclosure of an array of contracting performance failures. Based on the performance failures reporting table defined in the World Bank Framework for Disclosure in PPPs.
-
-#### Translation
-
-To translate the `name` and `description` fields, and to provide `documentationUrl` in multiple languages, add additional key-value pairs for each language, for example:
+## Example
 
 ```json
 {
-  "description": {
-    "es": "Descripción",
-    "en": "Description"
-  }
-}
-```
-
-## Codelists
-
-As in the [core standard repository](https://github.com/open-contracting/standard), the extension template also includes a codelists folder to store extension-specific codelists.
-
-Codelists are CSV files with camel case names, e.g. _implementationStatus.csv_. Be aware that a codelist in your extension using the same name as an existing codelist in the standard repository will override the existing codelist.
-
-Prefixing a codelist name with _+_ (e.g. _+milestoneStatus.csv_) indicates that contents of the codelist should be appended to an existing codelist of the same name. 
-
-## How the extensions work
-
-As of OCDS 1.1, the schema files from the core standard that extensions can extend are:
-
-* release-schema.json
-* record-package-schema.json
-* release-package-schema.json
-
-In the extension some or all of these files can be used to do a JSON Merge Patch of the corresponding file.
-
-A JSON Merge Patch is described by this [RFC](https://tools.ietf.org/html/rfc7386).
-
-The patches are very simple. They just copy the same structure from the core schema and allow you add your extra fields or update existing fields just in the places the extension wants to change the schema.
-
-Here are some simple examples of how this works. They are illustrative; any changes to existing fields should respect the [Conformance documentation](https://standard.open-contracting.org/latest/en/schema/conformance_and_extensions/).
-
-They all are examples of what could go in `release-schema.json` files in an extension.
-
-Add a new field `newField` to the release.
-
-```json
-{
-  "properties": {
-    "newField": {
-      "title": "New field",
-      "description": "A new field",
-      "type": "string"
+  "parties": [
+    {
+      "id": "ORG-0001",
+      "roles": [
+        "informationService"
```

Comment:
Including this here will help eForms users as this is also a field they have to fill in, but as this is from the EU extension and not this new extension I'm not sure it should be included, it could just confuse users?

Reply:
Hmm, I guess the question is should every organization reference in `Document.publisher` have 'informationService' in its roles or is that specific to eForms. Generally, we always add roles to match organization reference so I think we should probably add some usage guidance to that effect to the readme. @jpmckinney what do you think?

Reply:
Hmm, yes, I suppose it makes sense to add that code in this extension instead, with guidance. The code's description is sufficiently general. Good catch!

Reply:
Code added in https://github.com/open-contracting-extensions/ocds_document_publisher_extension/pull/1/commits/26e2570c72622962e674e422a40badc4c199cb46

Reply:
And removed from the EU extension in https://github.com/open-contracting-extensions/ocds_eu_extension/pull/12/commits/6ca87c22238deb1544f6c36f688f079a404d8102